jib-core/src/test/java/com/google/cloud/tools/jib/http/TestWebServer.java

```diff
@@ -35,7 +36,7 @@
 
   TestWebServer() throws IOException, InterruptedException {
     serverSocket = new ServerSocket(0);
-    executorService.submit(this::serve200);
+    ignoreReturn(executorService.submit(this::serve200));
```

Comment:
I'm kinda late here again but the [`@CanIgnoreReturnValue`](https://errorprone.info/api/latest/com/google/errorprone/annotations/CanIgnoreReturnValue.html) annotation can be an alternative:

```java
@CanIgnoreReturnValue
private static Future<Void> runOnExecutorService(Callable<Void> callable) {
  return executorService.submit(callable);
}
```